Common/cpp/Tools/JSScheduler.h

```diff
@@ -1,25 +1,52 @@
 #pragma once
 
 #include <ReactCommon/CallInvoker.h>
+#include <ReactCommon/RuntimeExecutor.h>
```

Comment:
Why was all of this moved from `.cpp` file to the header?

Reply:
The function was so tiny I found it an overkill of having a separate .cpp for it. However, if there's a reason to have a .cpp file I can restore it.

Reply:
if it works, then it's fine. I would just think if any of the imports can be not available somewhere else during build and before linking. But it seems fine so I will resolve it.